configs/messages.go

```diff
@@ -115,5 +115,15 @@ Follow https://launchpad.ethereum.org/ and happy staking!`
 		"https://github.com/NethermindEth/sedge/releases, with error:"
 	NeedVersionUpdate = "A new Version of sedge is available. Please update to the latest Version. See " +
 		"https://github.com/NethermindEth/sedge/releases for more information. Latest detected tag:"
-	VersionUpdated = "You are running the latest version of sedge. Version: "
+	VersionUpdated       = "You are running the latest version of sedge. Version: "
+	MnemonicPresentation = `
+This is your mnemonic:
+
+%s
+
+The mnemonic is the only way to get your deposit back, write it down now and keep it in a safe place.
+
+Press the <q> key when you have written down your mnemonic.
```

Comment:
Should this need to be updated for Windows and `notepad` @cbermudez97 ?

Reply:
Updated